util/configure

```diff
@@ -628,21 +628,12 @@ _END_
 
         {
             # check -msse4.2
-            my ($out, $cfile) = tempfile("resty-config-XXXXXX",
-                                         SUFFIX => '.c', TMPDIR => 1,
-                                         UNLINK => 1);
-
-            print $out "int main(void) { return 0; }";
```

Comment:
Tried to find something quickly when searching for solution. But I'm not a C-coder so I'm not comfortable doing that, since didn't find any easy way of doing that.